tests/regression/facebook/test_firefox_facebook_delete_post_club.sikuli/test_firefox_facebook_delete_post_club.py

```diff
@@ -16,7 +16,7 @@
 setAutoWaitTimeout(10)
 fb.wait_for_loaded()
 click(fb.club_post_marker.targetOffset(55, 75))
```

Comment:
Do we want to change all click()s to fb._click()s?
I think there are some missing ones. 

PS. offset for this fb.club_post_marker is (55, 75) but the one in array is (0, 0)